config/initializers/sidekiq.rb

```diff
@@ -1,22 +1,9 @@
-# Monkey patch `stringify_keys` to make rack 2.1.1 compatible with Sidekiq UI Admin panel.
-# Should be removed when 2.1.2 is released.
-# https://github.com/rack/rack/pull/1428
-module Rack
-  module Session
-    module Abstract
-      class SessionHash
-        private
-
-        def stringify_keys(other)
-          other.to_hash.transform_keys(&:to_s)
-        end
-      end
-    end
+Rails.application.reloader.to_prepare do
+  Dir.glob(Rails.root.join("lib/sidekiq/*.rb")).sort.each do |filename|
+    require_dependency filename
   end
 end
```

Comment:
this patch was included in rack, see https://github.com/rack/rack/pull/1428 - we don't need it anymore